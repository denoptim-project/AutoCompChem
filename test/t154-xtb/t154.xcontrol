$constrain
  angle: 5, 1, 7, auto
  angle: 5, 1, 8, auto
  angle: 7, 1, 8, auto
$end
$gfn
  method=2
  scc=true
  periodic=false
$opt
  logfile=t154_xtbtrj.xyz
  hessian=read
  optlevel=tight
$scc
  maxiterations=1000
$set
  runtyp opt
$chrg 0
$spin 0
