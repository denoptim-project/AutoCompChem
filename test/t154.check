#!/bin/bash

function not_passed() {
  echo "NOT Passed: check condition leading to line '$1' of '$0'"
  exit -1
}

if [ ! -f t154.log ] ; then not_passed $LINENO ; fi
if [ ! -f t154_allGeoms.xyz ] ; then not_passed $LINENO ; fi
if [ ! -f t154_lastGeom.sdf ] ; then not_passed $LINENO ; fi

# Only one geometry in the "trajectory" and XYZ format
n=0; n=$(grep -c "\$\$\$\$" t154_allGeoms.xyz)
if [ 0 != "$n" ];then not_passed $LINENO ; fi
n=0; n=$(grep -c "Ru" t154_allGeoms.xyz)
if [ 1 != "$n" ];then not_passed $LINENO ; fi

# Only one geometry in the final geometry with SDF format
n=0; n=$(grep -c "\$\$\$\$" t154_lastGeom.sdf)
if [ 1 != "$n" ];then not_passed $LINENO ; fi
n=0; n=$(grep -c "Ru" t154_lastGeom.sdf)
if [ 1 != "$n" ];then not_passed $LINENO ; fi
n=0; n=$(grep -c "> <SCF_ENERGY>" t154_lastGeom.sdf)
if [ 1 != "$n" ];then not_passed $LINENO ; fi

nrg=$(grep -A 1 "> <SCF_ENERGY>" t154_lastGeom.sdf | tail -n 1)
if [ "-21.621281660016" != "$nrg" ];then not_passed $LINENO ; fi

if grep -q 'Termination status: 0' "t154.log" ; then echo Passed ; exit 0 ; fi
not_passed $LINENO
exit -1
