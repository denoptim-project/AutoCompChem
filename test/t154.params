#
# Parameters for Automated Computational Chemistry
#
VERBOSITY: 6
TASK: AnalyseXTBOutput
JobOutputFile: ../t154-xtb/t154.out
SaveAllGeometries: XYZ t154_allGeoms.xyz
SaveLastGeometry: SDF t154_lastGeom.sdf
#TODO-gg: Not implemented yet
#SaveVibModes: 6 7 t154_vibModes_6_7.xyz
TEMPLATECONNECTIVITY: ../t154-tmpl.sdf
#GetEnergy: 
