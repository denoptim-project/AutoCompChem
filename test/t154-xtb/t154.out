      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.7.1 (edcfbbe) compiled by 'conda@afd1a870197c' on 2025-04-11

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,
     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,
     e01493. DOI: 10.1002/wcms.1493
   
   for GFN2-xTB:
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   for GFN1-xTB:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   for GFN0-xTB:
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   for GFN-FF:
   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.
     DOI: 10.1002/anie.202004239
   
   for ALPB and GBSA implicit solvation:
   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,
     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471
   
   for ddCOSMO and CPCM-X implicit solvation:
   * M. Stahn, S. Ehlert, S. Grimme, J. Phys. Chem. A,
     2023, XX, XXXX-XXXX. DOI: 10.1021/acs.jpca.3c04382
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.
     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.
     DOI: 10.1021/acsomega.9b02011
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   for SPH calculations refer to:
   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714
     DOI: 10.1021/acs.jctc.0c01306
   
   for ONIOM refer to:
   * C. Plett, A. Katbashev, S. Ehlert, S. Grimme, M. Bursch,
     Phys. Chem. Chem. Phys., 2023, 25, 17860-17868. DOI: 10.1039/D3CP02178E
   
   for DIPRO refer to:
   * J. Kohn, N. Gildemeister, S. Grimme, D. Fazzi, A. Hansen,
     J. Chem. Phys., 2023, just accepted.
   
   for PTB refer to:
   * S. Grimme, M. Mueller, A. Hansen, J. Chem. Phys., 2023, 158, 124111.
     DOI: 10.1063/5.0137838
   
   with help from (in alphabetical order)
   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher
   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. Hölzer
   A. Katbashev, J. Kohn, J. Koopman, C. Lavigne, S. Lehtola, F. März, M. Müller,
   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,
   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,
   J. Stückrath, T. Rose, and J. Unsleber
   
 * started run on 2025/06/24 at 22:39:25.468     
   ID    Z sym.   atoms
    1   44 Ru     1
    2    6 C      2, 5
    3    1 H      3, 4
    4    8 O      6
    5   17 Cl     7, 8
constraining angle 5 1 7 to   97.3462301°, actual value:   97.3462301°
constraining angle 5 1 8 to   97.4127018°, actual value:   97.4127018°
constraining angle 7 1 8 to  147.5530086°, actual value:  147.5530086°

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : xtb xtbopt.sdf --input t154.xcontrol --acc 0.05
          coordinate file            : xtbopt.sdf
          omp threads                :                    10

########################################################################
[WARNING] Please study the warnings concerning your input carefully
-1- set_rdsetbl: Set-blocks will become obsolete in xtb 6.0 and newer
########################################################################

           -------------------------------------------------
          |                 G F N 2 - x T B                 |
           -------------------------------------------------

        Reference                      10.1021/acs.jctc.8b01176
      * Hamiltonian:
        H0-scaling (s, p, d)           1.850000    2.230000    2.230000
        zeta-weighting                 0.500000
      * Dispersion:
        s8                             2.700000
        a1                             0.520000
        a2                             5.000000
        s9                             5.000000
      * Repulsion:
        kExp                           1.500000    1.000000
        rExp                           1.000000
      * Coulomb:
        alpha                          2.000000
        third order                    shell-resolved
        anisotropic                    true
        a3                             3.000000
        a5                             4.000000
        cn-shift                       1.200000
        cn-exp                         4.000000
        max-rad                        5.000000

q/qsh data taken from xtbrestart
CAMM data taken from xtbrestart

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  44          :
          :  # atomic orbitals                  41          :
          :  # shells                           17          :
          :  # electrons                        38          :
          :  max. iterations                  1000          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                       true          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    0.0500000          :
          :  -> integral cutoff          0.3801030E+02      :
          :  -> integral neglect         0.5000000E-09      :
          :  -> SCF convergence          0.5000000E-07 Eh   :
          :  -> wf. convergence          0.5000000E-05 e    :
          :  Broyden damping             0.4000000          :
          :  net charge                          0          :
          :  unpaired electrons                  0          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -21.7498324 -0.217498E+02  0.517E-05    1.62       0.0  T
   2    -21.7498324  0.112479E-10  0.455E-05    1.62     777.5  T
   3    -21.7498324  0.317542E-10  0.786E-05    1.62     449.6  T
   4    -21.7498324 -0.395630E-10  0.318E-05    1.62    1111.7  T

   *** convergence criteria satisfied after 4 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -1.0858078             -29.5463
       ...           ...                  ...                  ...
        13        2.0000           -0.4541353             -12.3577
        14        2.0000           -0.4520602             -12.3012
        15        2.0000           -0.4501800             -12.2500
        16        2.0000           -0.4496684             -12.2361
        17        2.0000           -0.4276207             -11.6362
        18        2.0000           -0.4217290             -11.4758
        19        2.0000           -0.4090726             -11.1314 (HOMO)
        20                         -0.3493999              -9.5077 (LUMO)
        21                         -0.3413238              -9.2879
        22                         -0.3402404              -9.2584
        23                         -0.3093555              -8.4180
        24                         -0.2826529              -7.6914
       ...                                ...                  ...
        41                          1.0427239              28.3740
      -------------------------------------------------------------
                  HL-Gap            0.0596727 Eh            1.6238 eV
             Fermi-level           -0.3792362 Eh          -10.3195 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.368 sec
 SCC setup                      ...        0 min,  0.000 sec (  0.031%)
 Dispersion                     ...        0 min,  0.000 sec (  0.012%)
 classical contributions        ...        0 min,  0.000 sec (  0.007%)
 integral evaluation            ...        0 min,  0.001 sec (  0.198%)
 iterations                     ...        0 min,  0.275 sec ( 74.682%)
 molecular gradient             ...        0 min,  0.092 sec ( 25.021%)
 printout                       ...        0 min,  0.000 sec (  0.045%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -21.621281660016 Eh    ::
         :: gradient norm              0.114939874766 Eh/a0 ::
         :: HOMO-LUMO gap              1.623776359325 eV    ::
         ::.................................................::
         :: SCC energy               -21.749815919416 Eh    ::
         :: -> isotropic ES            0.017456828251 Eh    ::
         :: -> anisotropic ES         -0.002520117195 Eh    ::
         :: -> anisotropic XC          0.003233911889 Eh    ::
         :: -> dispersion             -0.007462097685 Eh    ::
         :: repulsion energy           0.128534259400 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge              -0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -1.0858078             -29.5463
       ...           ...                  ...                  ...
         7        2.0000           -0.6345447             -17.2668
         8        2.0000           -0.5589433             -15.2096
         9        2.0000           -0.5311161             -14.4524
        10        2.0000           -0.4866173             -13.2415
        11        2.0000           -0.4677597             -12.7284
        12        2.0000           -0.4637267             -12.6186
        13        2.0000           -0.4541353             -12.3577
        14        2.0000           -0.4520602             -12.3012
        15        2.0000           -0.4501800             -12.2500
        16        2.0000           -0.4496684             -12.2361
        17        2.0000           -0.4276207             -11.6362
        18        2.0000           -0.4217290             -11.4758
        19        2.0000           -0.4090726             -11.1314 (HOMO)
        20                         -0.3493999              -9.5077 (LUMO)
        21                         -0.3413238              -9.2879
        22                         -0.3402404              -9.2584
        23                         -0.3093555              -8.4180
        24                         -0.2826529              -7.6914
        25                          0.0152171               0.4141
        26                          0.0228530               0.6219
        27                          0.0228744               0.6224
        28                          0.0239368               0.6514
        29                          0.0242060               0.6587
        30                          0.0242957               0.6611
       ...                                ...                  ...
        41                          1.0427239              28.3740
      -------------------------------------------------------------
                  HL-Gap            0.0596727 Eh            1.6238 eV
             Fermi-level           -0.3792362 Eh          -10.3195 eV

     #   Z          covCN         q      C6AA      α(0)
     1  44 Ru       3.493    -0.045   317.241    37.729
     2   6 C        2.774    -0.050    29.224     8.854
     3   1 H        0.924     0.125     1.574     1.962
     4   1 H        0.925     0.132     1.522     1.929
     5   6 C        1.770     0.224    25.981     8.241
     6   8 O        0.856    -0.047    15.754     5.321
     7  17 Cl       0.823    -0.169   104.372    15.629
     8  17 Cl       0.823    -0.169   104.370    15.629

 Mol. C6AA /au·bohr⁶  :       3000.339495
 Mol. C8AA /au·bohr⁸  :     152976.108285
 Mol. α(0) /au        :         95.293321


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom

 ---------------------------------------------------------------------------
     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO
 ---------------------------------------------------------------------------
     1  44 Ru   6.070 --     2 C    1.982     5 C    1.312     8 Cl   1.128
                             7 Cl   1.128     6 O    0.278     4 H    0.149
     2   6 C    3.943 --     1 Ru   1.982     3 H    0.873     4 H    0.812
                             8 Cl   0.130     7 Cl   0.130
     3   1 H    0.984 --     2 C    0.873
     4   1 H    0.983 --     2 C    0.812     1 Ru   0.149
     5   6 C    3.659 --     6 O    2.279     1 Ru   1.312
     6   8 O    2.611 --     5 C    2.279     1 Ru   0.278
     7  17 Cl   1.417 --     1 Ru   1.128     2 C    0.130
     8  17 Cl   1.417 --     1 Ru   1.128     2 C    0.130
 ---------------------------------------------------------------------------

Topologies differ in bond orders
Writing corrected topology to xtbtopo.sdf

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.762       0.001       1.126
   full:       -0.236       0.001       0.553       1.528
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:        2.520       0.003      -7.332      -3.934      -0.002       4.812
  q+dip:        4.349       0.001     -10.745      -2.453      -0.002       6.396
   full:        2.881       0.001      -9.637      -1.916      -0.003       6.756

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :      214.0075735
   center of mass at/Å    :       -0.2960009      -0.0002076      -0.6539783
  moments of inertia/u·Å² :        0.2775188E+03   0.4189156E+03   0.5599353E+03
rotational constants/cm⁻¹ :        0.6074410E-01   0.4024112E-01   0.3010639E-01

 * 5 selected distances

     #   Z          #   Z                                           value/Å
Note: The following floating-point exceptions are signalling: IEEE_INVALID_FLAG IEEE_UNDERFLOW_FLAG IEEE_DENORMAL
normal termination of xtb
     1  44 Ru       2   6 C                                       1.7815002
     2   6 C        3   1 H                                       1.0960350 (min)
     5   6 C        6   8 O                                       1.1517734
     1  44 Ru       7  17 Cl                                      2.3160852 (max)
     1  44 Ru       8  17 Cl                                      2.3160724

 * 4 distinct bonds (by element types)

   Z      Z             #   av. dist./Å        max./Å        min./Å
   1 H    6 C           1     1.0960350     1.0960350     1.0960350
   6 C    8 O           1     1.1517734     1.1517734     1.1517734
   6 C   44 Ru          1     1.7815002     1.7815002     1.7815002
  17 Cl  44 Ru          2     2.3160788     2.3160852     2.3160724


optimized geometry written to: xtbopt.sdf


           -------------------------------------------------
          | TOTAL ENERGY              -21.621281660016 Eh   |
          | GRADIENT NORM               0.114939874766 Eh/α |
          | HOMO-LUMO GAP               1.623776359325 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2025/06/24 at 22:39:25.953     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.484 sec
 *  cpu-time:     0 d,  0 h,  0 min,  6.093 sec
 * ratio c/w:    12.582 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.368 sec
 *  cpu-time:     0 d,  0 h,  0 min,  4.659 sec
 * ratio c/w:    12.657 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.000 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.000 sec
 * ratio c/w:       NaN speedup

